src/pages/Landing.tsx

```diff
@@ -0,0 +1,140 @@
+      <motion.section
+        className="scroll-mt-16 bg-gray-50 py-20"
+        initial={{ opacity: 0 }}
+        whileInView={{ opacity: 1 }}
+        transition={{ duration: 0.8 }}
+        viewport={{ once: true }}
+        id="what-is-section"
+        data-testid="what-is-section"
+      >
+        <div className="mx-auto max-w-4xl space-y-16 px-4">
+          <div>
+            <h2
+              id="what-are-feature-flags"
+              className="mb-6 text-3xl font-bold text-[#041484] md:text-4xl"
+            >
+              What are Feature Flags?
+            </h2>
+            <p className="text-lg text-gray-600">{WHAT_IS_FEATURE_FLAG}</p>
+          </div>
+
+          <div data-testid="why-use-section">
+            <h2
+              id="why-use-feature-flags"
+              className="mb-6 text-3xl font-bold text-[#041484] md:text-4xl"
+            >
+              Why Use Feature Flags?
+            </h2>
+            <ul
+              className="list-inside list-disc space-y-3 text-lg text-gray-600"
+              aria-label="Benefits of using feature flags"
+            >
+              {WHY_USE_FEATURE_FLAGS.map((item) => (
+                <li key={item}>{item}</li>
+              ))}
+            </ul>
+          </div>
+        </div>
+      </motion.section>
+      <motion.section
+        className="py-20"
+        initial={{ opacity: 0 }}
+        whileInView={{ opacity: 1 }}
+        transition={{ duration: 0.8 }}
+        viewport={{ once: true }}
+        data-testid="features-section"
+      >
+        <div className="mx-auto max-w-4xl px-4">
+          <h2 className="mb-12 text-center text-3xl font-bold text-[#041484] md:text-4xl">
+            Key Features
+          </h2>
+          <div
+            className="grid gap-8 md:grid-cols-2"
+            data-testid="features-grid"
+          >
+            {FEATURES.map((feature, index) => (
+              <motion.div
+                key={feature.title}
+                className="rounded-lg border border-gray-100 bg-white p-6 shadow-lg transition-all hover:shadow-xl"
+                initial={{ opacity: 0, y: 20 }}
+                whileInView={{ opacity: 1, y: 0 }}
+                transition={{ delay: index * 0.1 }}
+                viewport={{ once: true }}
+                data-testid={`feature-card-${index}`}
+              >
+                <h3 className="mb-3 text-xl font-semibold text-[#041484]">
+                  {feature.title}
+                </h3>
+                <p className="text-gray-600">{feature.description}</p>
+              </motion.div>
+            ))}
+          </div>
+        </div>
+      </motion.section>
+      <motion.footer
+        className="bg-gradient-to-r from-primary to-primary-light py-12 text-white"
+        initial={{ opacity: 0 }}
+        whileInView={{ opacity: 1 }}
+        transition={{ duration: 0.8 }}
+        viewport={{ once: true }}
+        role="contentinfo"
+        data-testid="footer"
```

Comment:
can we consolidate the `framer-motion` animation logic into a utility to avoid repetition

Reply:
Sure